lib/openhab/dsl/items/rollershutter_item.rb

```diff
@@ -17,13 +17,7 @@ class RollershutterItem < Numeric
 
         def_delegator :@rollershutter_item, :to_s
 
-        java_import Java::OrgOpenhabCoreLibraryTypes::PercentType
-        java_import Java::OrgOpenhabCoreLibraryTypes::UpDownType
-        java_import Java::OrgOpenhabCoreLibraryTypes::StopMoveType
-
-        item_command Java::OrgOpenhabCoreLibraryTypes::StopMoveType
-        item_command Java::OrgOpenhabCoreLibraryTypes::UpDownType
-        item_state   Java::OrgOpenhabCoreLibraryTypes::UpDownType
+        oh_item_type Java::OrgOpenhabCoreLibraryItems::RollershutterItem
```

Comment:
Should we just call this `item_type` instead of `oh_item_type`?

Reply:
I agree.. nothing else has oh_ in front of it. 

item_type doesn't really make it clear to me based on its name what it is doing either. 
to be in line with other ruby stuff they should all probably be renamed
def_item_command
def_item_state
def_item_from_types  (?maybe?)

Reply:
def_item_from_type ?

Reply:
Was a bit unsure what to name it, but item_type feels best IMO. Am a bit busy this weekend, so won't have much time to make adjustments, but hopefully on Monday.